app/jobs/streams/twitch_webhook_registration_job.rb

```diff
@@ -0,0 +1,29 @@
+module Streams
+  class TwitchWebhookRegistrationJob < ApplicationJob
```

Comment:
Can do! I haven't used explicit named queues for each job before, can you explain a bit about why that is done?

Reply:
Hi! For what I've gathered it's for two reasons. One quite generic: having tens of different types of jobs in the same queue makes it hard to allocate more resources to a task or to change prioritities on the tasks. The second one is because it makes monitoring easier: you can look at the web monitor and instantly know if a task is swamped or if a task is being processed. 

If everything is in `:default` it becomes really hard to understand what's going on in the queing system